types/chrome/index.d.ts

```diff
@@ -7626,7 +7626,7 @@ declare namespace chrome {
          */
         export function getKeyPair(
             certificate: ArrayBuffer,
-            parameters: Object,
+            parameters: object,
```

Comment:
Honestly, the cryptography stuff is where things start going over my head. `parameters` is a JSON object, it looks like we could make an interface for this (see [docs](https://developer.chrome.com/docs/extensions/reference/api/platformKeys#method-getKeyPair)), but I'm not gonna be the one. My suggestion is a key-unknown object.